### DISCUSSION TITLE
Azure Sandbox Activation Problem!

### DISCUSSION THREAD

**Discussion by iamshivampakade** [github/india#366] [Students]
I am unable to activate sandbox for the performing exercises given in module, when I click on Activate Sandbox button the previous sandbox gets deactivated and this happening to me repeatedly and my all sandboxes get deactivated please help
 @ey-mentor1 , @ey-mentor2 , @EY-Mentor3 , @EY-Mentor4

**Comment by divyavemasani2910** [comment]
Hey there! wave

If you are also a Tech-Enthusiast and looking for a One-Stop place to collaborate & get notified for various Hackathons + Daily Events & Workshops + Internships/Jobs + Free Scholarship, Courses, Swags & Goodies + 24/7 Live Chat & Mentors to help you with your Doubts + 24/7 Live Music  and much more...then join the community through the link below star_struck

With over more than 2k+ members from more than 20+ different countries we are one of the most active growing community partying_face fire fire
Link: https://discord.gg/xjfP5796
Join this channel there we assist to clear your doubts
